static/solo/js/menu.js

```diff
@@ -0,0 +1,225 @@
+;(function(w, d) {
```

Comment:
I suggest continue using `window` and `document` to avoid confusions.
`w` and `d` are too short, we will end up minifying this code anyways.

We do this because we want to convert global variables (document, window, etc) to local variables, creating scope with the immediate function.

Is quicker/performant to use local variables instead of globals.